tests/db_test.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import psycopg2
+
+
+def test_users_table_rows_count(cursor, build_users_table):
```

Comment:
Where is this cursor defined? The only place I see it is in conftest.py and this script doesn't import that one.
It also might make sense to have one database test file and not split it up.

Reply:
Also, if I understood the code in the conftest.py correctly, you always call rollback, which erases all of your changes. So if you are expecting the data to be here because of that execution, it might not be.

Reply:
> Where is this cursor defined? The only place I see it is in conftest.py and this script doesn't import that one.
> It also might make sense to have one database test file and not split it up.

The cursor is defined in conftest.py file that aggregate all the fixtures in one place. I tried to keep the db_test.py file short.

Reply:
> Also, if I understood the code in the conftest.py correctly, you always call rollback, which erases all of your changes. So if you are expecting the data to be here because of that execution, it might not be.

I call rollback after the cursor finish his job so the data still in the DB in this point. This test was passed.